project/plugins.sbt

```diff
@@ -2,7 +2,9 @@ logLevel := Level.Info
 
 resolvers += "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/"
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.3.8")
+libraryDependencies += "com.typesafe.play" %% "anorm" % "2.4.0"
```

Comment:
i'll try it in build.sbt.  i was mucking around for ages after upping to 2.5 and it worked there